Sources/Plasma/PubUtilLib/plAvatar/plPhysicalControllerCore.cpp

```diff
@@ -785,9 +785,8 @@ static float AngleRad2d( float x1, float y1, float x3, float y3 )
 
         if ( value < 0.0 )
         {
-            value = (float)(value + TWO_PI);
+            value += (float)(value + hsConstants::two_pi<double>);
```

Comment:
Looks like an unintentional behavior change... Also, is the cast needed?
```suggestion
            value += hsConstants::two_pi<float>;
```

Reply:
There were a few cases where math was done in `double` space, so I tried to leave them that way.  It's unclear which parts actually need the extra precision, and which are fine with single precision...

In this particular case, I probably changed it and then changed it back (and forgot to remove the `+`)...  However, looking at it again, I think in this particular case it's fine to just convert it to using purely floats.